packages/cashier/src/Components/recent-transaction/index.js

```diff
@@ -0,0 +1,4 @@
+import RecentTransaction from './recent-transaction.jsx';
```

Comment:
Could you follow the current folders structure of the project for `recent-transaction`, which means remove the `recent-transaction` folder and the `index.js` file, move the `recent-transaction.jsx` to component folder and move the `recent-transaction.scss` to `Sass` folder.

Reply:
e everyone has a different view on that, I had it as a separate file with the styles in the main scss file, but after first review I was told to move it into a separate folder like I have now, heh

but okay

Reply:
I see, I will talk about it in our catch-up today, but please change this now